src/controllers/test.js

```diff
@@ -0,0 +1,9 @@
+const { Test } = require("../models/test");
```

Comment:
I don't like that we have own folders for controllers, models and routes. Can you please restructure it according to this [page](https://stackoverflow.com/questions/60056949/in-nodejs-app-how-to-use-dotenv-and-config)

Reply:
I don`t undestand what you means. It`s page, where show how use dotenv better.

Reply:
My bad. I meant this [one](https://github.com/goldbergyoni/nodebestpractices/blob/master/sections/projectstructre/breakintcomponents.md)

Reply:
Okay, a try do. Check please telegram.

Reply:
/+/